commons/build.gradle

```diff
@@ -25,4 +25,6 @@ dependencies {
     // For now taking the version from kafka-clients
     compileOnly "com.github.luben:zstd-jni:$zstdVersion"
     testImplementation "com.github.luben:zstd-jni:$zstdVersion"
+    implementation group: 'commons-io', name: 'commons-io', version: '2.11.0'
+    testRuntimeOnly "org.slf4j:slf4j-log4j12:1.7.36"
```

Comment:
Should this be in a root Gradle file since it's used in both modules?

Reply:
Yes, done